ihatemoney/web.py

```diff
@@ -646,6 +646,33 @@ def list_bills():
     )
 
 
+@main.route("/<project_id>/archived")
```

Comment:
I also feel that this is a lot of copy/pasting. You probably want to remove some code that's not needed here (no need to support the addition of new bills), we probably should also remove the possibility to add a bill when we're browing the archives.